### PR TITLE
Add `apt` Snippet In Command Library

### LINKED ISSUE #512: Add 'apt' to Command Library
**Description**
Add 'apt' to snippets.yml to allow detection of Layer changes made using the 'apt' command.

**Implementation**
Duplicate the snippets.yml entry for 'apt-get'

**References**
tern/command_lib/snippets.yml, lines 9-14

**Super Issues**
n.A.

### PR DESCRIPTION
**Pull request by HeroicHitesh**
Add `apt` command to `snippets.yml` present under `command_lib`

Resolves #512

Signed-off-by: @HeroicHitesh hiteshkumar_1mv17cs042@sirmvit.edu

### CODE REVIEW

File: tern/command_lib/snippets.yml [hunk 1]
```diff
@@ -11,15 +11,20 @@ apt-get:
     - 'install' # subcommand to install
   remove:
     - 'purge' # subcommand to remove a package
+    ignore: # list of subcommands that don't add or remove packages
+    - 'update' 
+  packages: 'dpkg' # refer to base.yml's method of collection
+
+apt:
+  install: 'install' # subcommand to install
+  remove: 'purge' # subcommand to remove a package
```
Comment (nishakm) [inline tern/command_lib/snippets.yml:20]: You could just copy exactly what is under `apt-get` and you should be fine.

**Review by nishakm — CHANGES_REQUESTED** [review]
Hello @HeroicHitesh, you will need a real email address to submit a PR to this project :). I would suggest following the CONTRIBUTING.md guidelines to submit a PR.

File: tern/command_lib/snippets.yml (hunk 1 above)
Comment (HeroicHitesh) [reply tern/command_lib/snippets.yml]: Yeah, actually I did that but I think `prettier` did this😅
Updated again.

**Comment by HeroicHitesh** [comment]
> Hello @HeroicHitesh, you will need a real email address to submit a PR to this project :). I would suggest following the CONTRIBUTING.md guidelines to submit a PR.

Actually I read the CONTRIBUTING.md and also, hiteshkumar_1mv17cs042@sirmvit.edu is my real email address, not sure what's the issue. Can you please explain the issue in detail?

**Comment by nishakm** [comment]
@HeroicHitesh sorry for getting back to you so late. The commit message should be like this example: https://github.com/tern-tools/tern/blob/master/CONTRIBUTING.md#commit-message-format. Also, `snippets.yml` has now been moved to `tern/analyze/default/command_lib/snippets.yml`. You may close this PR and open a new one with your change. Thank you for contributing!

**Comment by HeroicHitesh** [comment]
> You may close this PR and open a new one with your change. Thank you for contributing!

@nishakm I have updated the same PR. Is it fine? or do I need to close it and create new PR?

**Review by nishakm — APPROVED** [review]

**Comment by nishakm** [comment]
@HeroicHitesh Thank you!

**Pull request merged**